src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3826,20 +3826,24 @@
      * member value pair.
      *
      * <p>For example:</p>
-     *
      * <pre>
-     *     { 1, 2 }
+     * &#64;Annotation({1, 2})
      * </pre>
      *
```

Comment:
minor request: this line can be also removed

Reply:
@timurt I did that but one check was failing then I leave this line blank.

Reply:
here is example https://github.com/checkstyle/checkstyle/pull/9378/files

Lets try again and see what will fail